shared/js/initApp.js

```diff
@@ -21,6 +21,11 @@ var baseOptions = {
     if (appOptions.level.isProjectLevel) {
       return;
     }
+    if (appOptions.channel) {
+      // Don't send the levelSource to Dashboard for channel-backed levels.
+      // (The levelSource is already stored in the channels API.)
+      delete report.program;
```

Comment:
Maybe it would be better to keep a list of things we **do** want to send back to the server?  What is the minimal payload to create an `Activity` record?